inference-gateway/docker/Dockerfile.postgres

```diff
@@ -1,7 +1,8 @@
 FROM quay.io/tembo/tembo-local
 
-RUN trunk install pg_timeseries --version 0.1.5
+RUN trunk install pg_timeseries --version 0.1.6
 RUN trunk install pg_cron --version 1.6.2
+RUN trunk install tembo_ivm --version 1.9
```

Comment:
let's keep this at v 0.1.5, unless we have a plan to upgrade the inference service's dev/prod environments to the latest version?